runners/direct-java/src/main/java/org/apache/beam/runners/direct/TransformExecutor.java

```diff
@@ -92,7 +92,7 @@ private TransformExecutor(
 
   @Override
   public void run() {
-    MetricsContainer metricsContainer = new MetricsContainer(transform.getFullName());
+    MetricsContainerImpl metricsContainer = new MetricsContainerImpl(transform.getFullName());
```

Comment:
This can be referred to as the interface, yes? (feel free to file a JIRA to cleanup instead of doing so now.)